apps/studio/src/pages/dashboard/index.tsx

```diff
@@ -1,17 +1,281 @@
+  BiHome,
+  BiSolidCircle,
+} from 'react-icons/bi'
+import { useEffect, useState } from 'react'
+import { boolean } from 'zod'
+import _ from 'lodash'
+import { MdOutlineHorizontalRule } from 'react-icons/md'
 
-export default function Dashboard() {
-  const { isOpen, onOpen, onClose } = useDisclosure()
+const Dashboard: NextPageWithLayout = () => {
+  const {
+    isOpen: isPageCreateModalOpen,
+    onOpen: onPageCreateModalOpen,
+    onClose: onpageCreateModalClose,
+  } = useDisclosure()
+  return (
+    <VStack bgColor="#F3F5F7" w="100%" p="1.75rem" minH="100vh">
+      <Text
+        alignSelf="flex-start"
+        textColor="base.content.default"
+        textStyle="h5"
+      >
+        My Pages
+      </Text>
+      <HStack w="100%" alignItems="end">
+        <Text
+          alignSelf="flex-start"
+          mr="auto"
+          textColor="base.content.default"
+          textStyle="body-2"
+        >
+          Double click a page to start editing.
+        </Text>
+        <Button alignSelf="flex-end" ml="auto" variant="outline" size="xs">
+          Create a folder
+        </Button>
+        <Button onClick={onPageCreateModalOpen} alignSelf="flex-end" size="xs">
+          Create a new page
+        </Button>
+      </HStack>
+      <DashboardTable />
+      <PageCreateModal
+        isOpen={isPageCreateModalOpen}
+        onClose={onpageCreateModalClose}
+      />
+    </VStack>
+  )
+}
+
+export const DashboardTable = (): JSX.Element => {
+  const dummyChildData: {
+    id: string
+    name: string
+    permalink: string
+    type: 'page' | 'folder'
+    status: 'folder' | 'draft' | 'published'
+    lastEditUser: string
+    lastEditDate: Date | 'folder'
+  }[] = [
+    {
+      id: '0001',
+      name: 'Test Page 1',
+      permalink: '/',
+      type: 'page',
+      status: 'draft',
+      lastEditUser: 'user1@test.com',
+      lastEditDate: new Date(),
+    },
+    {
+      id: '0003',
+      name: 'Test Folder 1',
+      permalink: '/testfolder1',
+      type: 'folder',
+      status: 'folder',
+      lastEditUser: 'folder',
+      lastEditDate: 'folder',
+    },
+    {
+      id: '0002',
+      name: 'Test Page 2',
+      permalink: '/testpage2',
+      type: 'page',
+      status: 'published',
+      lastEditUser: 'user2@test.com',
+      lastEditDate: new Date(50000000000),
+    },
+    {
+      id: '0004',
+      name: 'Test Folder 2',
+      permalink: '/testfolder2',
+      type: 'folder',
+      status: 'folder',
+      lastEditUser: 'folder',
+      lastEditDate: 'folder',
+    },
+  ]
+
+  const [pageNumber, onPageChange] = useState(1)
+  const [dataToDisplay, setDataToDisplay] = useState(dummyChildData)
 
+  const entriesPerPage = 6
   return (
-    <div>
-      {/* TODO: Dashboard layout */}
-      <h1>DASHBOARD</h1>
+    <>
+      <TableContainer
+        w="100%"
+        borderRadius="8px"
+        border="1px"
+        borderColor="base.divider.medium"
+        bgColor="#FFF"
+      >
+        <Table>
```

Comment:
Usually we keep the stuff under the `pages` directory lean. It should mostly import components from the root components folder or the `/features` folder

Suggestion is to move this table layout under the `/features/dashboard` folder

Reply:
Understand the above, will shift the layout file to /features/dashboard